x/tokenfactory/types/params.go

```diff
@@ -1,25 +1,49 @@
 package types
 
 import (
+	"fmt"
+
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
-// ParamTable for tokenfactory module.
+const DenomAllowListMaxSize = 2000
```

Comment:
DefaultDenomAllowListMaxSize? To make it clearer that this is just the default and might be changed by future gov proposals

Reply:
renamed and added comment